src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/ResidentialChecksServiceTest.kt

```diff
@@ -118,7 +124,7 @@ class ResidentialChecksServiceTest {
 
     whenever(assessmentService.getCurrentAssessment(PRISON_NUMBER)).thenReturn(assessmentEntity)
     whenever(addressService.getCurfewAddressCheckRequest(ADDRESS_REQUEST_ID, PRISON_NUMBER)).thenReturn(
-      aStandardAddressCheckRequest(),
+      aStandardAddressCheckRequest(true, true, true, true, true, true),
```

Comment:
loads of booleans like this are difficult to understand.. you could add an override in the test data for `aStandardAddressCheckRequestWithAllChecksComplete`? 

And then places where they aren't all true it'd be better to at least use named parameters